src/fns.js

```diff
@@ -0,0 +1,52 @@
+export const shallowMerge = (...objects) => {
```

Comment:
This file is small now but it might grow in the future as we add more features.  It might be good to split this file up a bit more (either now or in a future PR). fns.js is a bit generic and I think we could make these files a bit more specific.  

You could create a util folder, then make files like object.js (for shallow merge, flatten, range, maybe getAttributes), something like time.js? (for parseDuration, you could also export the constants in that function if you wanted to).  This would also give a place for future util-ish files to go and might prevent the file from getting out of hand.

Reply:
Also if you wanted to reference one util file when importing you could put a file there like util.js or index.js that imports files in that folder.